doc/classes/Quaternion.xml

```diff
@@ -71,7 +71,7 @@
 			<param index="0" name="to" type="Quaternion" />
 			<description>
 				Returns the angle between this quaternion and [param to]. This is the magnitude of the angle you would need to rotate by to get from one to the other.
-				[b]Note:[/b] This method has an abnormally high amount of floating-point error, so methods such as [code]is_zero_approx[/code] will not work reliably.
+				[b]Note:[/b] This method has an abnormally high number of floating-point errors, so methods such as [code]is_zero_approx[/code] will not work reliably.
```

Comment:
This change is not correct. There is only one returned value, so only one value to have floating-point error. This documentation is about the magnitude of that error. The old text was correct.

Reply:
I read it as "this method frequently produces floating-point errors", so I think this needs to be re-worked because the previous version is confusing.

> The magnitude of the floating-point error for this method is abnormally high, so...

Is that what it means? If not, can you suggest an alternative wording?

Reply:
> The magnitude of the floating-point error for this method is abnormally high, so...

This is correct and even better than the old text. If you submit a PR I will approve. Thanks in advance!